.gitignore

```diff
@@ -1,3 +1,3 @@
-build/
+*build/
```

Comment:
Why the wildcard? It might affect deeper directories that have similar names.

Reply:
Because in my local dev environment `source/_build` was created. This change would cover all possible variations and given the goal of the documentation a `*build` directory that is part of the source seemed unlikely. But I am happy to make the `.gitignore` more explicit.